capella_model_explorer/components.py

```diff
@@ -302,6 +307,19 @@ def report_loader(
             template_id=template_id,
             model_element_uuid=model_element_uuid,
         ),
+        hx_headers=(
+            "{"
+            '"capella-model-explorer-version":'
+            f'"{capella_model_explorer.__version__}",'
+            f'"capellambse-version": "{capellambse.__version__}",'
+            '"capellambse-context-diagrams-version":'
+            f'"{capellambse_context_diagrams.__version__}",'
+            '"template-file-hash":'
+            f'"{core.compute_file_hash(str(template.path))}",'
+            '"model-revision":'
+            f'"{state.model.info.resources["\x00"].rev_hash}"'
+            "}"
```

Comment:
This is hard to read. I'd propose to create Python dictionary and dump it with the json built-in library.